src/index.js

```diff
@@ -0,0 +1,62 @@
+
+
+// let count = 0;
+// const tempUp = document.getElementById("increment");
+// const tempDown = document.getElementById("decrement");
+// const textHolder = document.getElementById("tempCount");
+// textHolder.innerHTML = count;
+
+// tempUp.addEventListener("click", function() {
+//   textHolder.innerHTML = ++count;
+// });
+
+// tempDown.addEventListener("click", function() {
+//   textHolder.innerHTML = --count;
+// });
+
+const state = {count: 0,};
+const tempCounter = () => {
+
+    const tempUp = document.getElementById("increment");
+    const tempDown = document.getElementById("decrement");
+    const textHolder = document.getElementById("tempCount");
+    textHolder.innerHTML = state.count;
+
+    tempUp.addEventListener("click", function() {
+    textHolder.innerHTML = ++state.count;
```

Comment:
Just suggested indentation.  This works except that you have a `temperature` tag in the HTML doc which isn't valid HTML.

```suggestion
      textHolder.innerHTML = ++state.count;
```